types/tests/transaction_test.go

```diff
@@ -0,0 +1,154 @@
+package tests
+
+import (
+	"math/big"
+	"testing"
+
+	"github.com/0xPolygon/polygon-edge/crypto"
+
+	"github.com/0xPolygon/polygon-edge/types"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/umbracle/ethgo"
+	"github.com/umbracle/ethgo/wallet"
+)
+
+func getUint64FromBigInt(b *ethgo.ArgBig) (uint64, bool) {
+	g := (*big.Int)(b)
+	if !g.IsUint64() {
+		return 0, false
+	}
+	return g.Uint64(), true
+}
+
+func TestTransactions(t *testing.T) {
+	var transactions []struct {
+		Name              string         `json:"name"`
+		AccountAddress    ethgo.Address  `json:"accountAddress"`
+		PrivateKey        ethgo.ArgBytes `json:"privateKey"`
+		SignedTransaction ethgo.ArgBytes `json:"signedTransactionChainId5"`
+
+		Data     *ethgo.ArgBytes `json:"data,omitempty"`
+		Value    *ethgo.ArgBig   `json:"value,omitempty"`
+		To       *ethgo.Address  `json:"to,omitempty"`
+		GasLimit *ethgo.ArgBig   `json:"gasLimit,omitempty"`
+		Nonce    uint64          `json:"nonce,omitempty"`
```

Comment:
Do we need to omit zero value Nonce and Name?

Reply:
No needed atm. Removed these tests.